SU-SWS/react_paragraphs
# D8CORE-2857 keep upper case characters in link fields

# READY FOR REVIEW

# Summary
- Keep upper case characters in link fields
- Use the help text as html so that links, `em` `strong` and other tags will display.
- Fixed the display and the logic for required select fields.

# Need Review By (Date)
- 10/5

# Urgency
- high

# Steps to Test
1. checkout this branch and clear browser cache
1. create a page with a card or other paragraph with a link field
1. put in a url in the link field with some upper case and lower case characters.
1. save and verify the upper case characters are still there as upper case.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
